src/RenderWebGL.js

```diff
@@ -243,6 +243,21 @@ class RenderWebGL extends EventEmitter {
         this._debugCanvas = canvas;
     }
 
+    /**
+     * Force "touching color" operations to use (or not use) the GPU. By default the renderer will decide.
+     * @param {boolean} forceGPU - true to force the renderer to use the GPU, false to force CPU.
+     */
+    setForceGPU (forceGPU) {
```

Comment:
Maybe use an enum instead? One that accounts for the three options? Like `ONLY_GPU` `MIXED_CPU_GPU` `ONLY_CPU`?